bot/exts/evergreen/avatar_modification/avatar_modify.py

```diff
@@ -121,6 +121,50 @@ async def eightbit_command(self, ctx: commands.Context) -> None:
 
         await ctx.send(embed=embed, file=file)
 
+    @avatar_modify.group(name="reverse", root_aliases=['reverse'])
+    async def reverse(self, ctx: commands.Context) -> None:
+        """Group for the reverse commands."""
+        if not ctx.invoked_subcommand:
+            await invoke_help_command(ctx)
+
+    @reverse.command(name="text")
+    async def reverse_text(self, ctx: commands.Context, *, text: str) -> None:
+        """Sends the given text backwards."""
+        await ctx.send(f"> {text[::-1]}", allowed_mentions=discord.AllowedMentions.none())
+
+    @reverse.command(name="image")
+    async def reverse_photo(self, ctx: commands.Context) -> None:
+        """
+        Sends a reversed version of the users profile picture.
+
+        If an image is attached, the given image will be flipped.
+        """
+        async with ctx.typing():
+            user = await self._fetch_user(ctx.author.id)
+            if not user:
+                await ctx.send(f"{Emojis.cross_mark} Could not get user info.")
+                return
+
+            image_bytes = await user.avatar_url_as(size=1024).read()
+            filename = file_safe_name("reverse_avatar", ctx.author.display_name)
+
+            file = await in_executor(
+                PfpEffects.apply_effect,
+                image_bytes,
+                PfpEffects.flip_effect,
+                filename
+            )
+
+            embed = discord.Embed(
+                title="Your reversed avatar.",
+                description="Here is your reversed avatar. I think it is a spitting image of you."
```

Comment:
Did you mean `splitting`?

Reply:
`spitting image` is correct, https://www.merriam-webster.com/dictionary/spitting%20image. `splitting image` would be a great pun for the `.mosaic` command though, thinking about it 😄